static/static_file_modifier.go

```diff
@@ -57,10 +61,19 @@ func (s *staticModifier) ModifyRequest(req *http.Request) error {
 }
 
 // ModifyResponse reads the file rooted at rootPath joined with the request URL
-// path.  In the case that the file cannot be found, the response will be a 404.
+// path. In the case that the the request path is a key in s.explicitPaths, ModifyRequest
+// will attempt to open the file located at s.rootPath joined by the value in s.explicitPaths
+// (keyed by res.Request.URL.Path). In the case that the file cannot be found, the response
+// will be a 404.
```

Comment:
Clarify comment about what happens when the file is in explicitPaths, but can't be found on the filesystem at that path, but does exist at rootPath (which is still a 404, I believe, based on code)